builders/interface.go

```diff
@@ -39,10 +39,3 @@ type VMQueryBuilder interface {
 
 	IsInterfaceNil() bool
 }
-
-// TxSigner defines the method used by a struct used to create valid signatures
-type TxSigner interface {
```

Comment:
consider moving this back:
the interface definition should be where used

Reply:
done